src/app/organizer/ItemTable.tsx

```diff
@@ -628,48 +635,32 @@ function ItemTable({
         }
       : undefined;
 
+  const onMoveSelectedItems = (store) => {
+    const items = selectedFlatRows?.map((d) => d.original);
+    for (const item of items) {
+      moveItemTo(item, store, false, item.amount);
```

Comment:
A better way to do this is to generate a loadout and apply the loadout. That'll do things like making sure that one of the moved items doesn't get moved right back to the vault to make way for the next one. There are a few examples of doing this elsewhere (for example, the command to move a search).

Reply:
One weird thing that is happening now (can't remember if it was happening before the loadout thing but it probably was) is that selected rows are being changed once stuff has been moved. For example if I select my ace of spades and move it, afterwards either an Austringer I have or an Erianna's Vow is selected. I haven't chased down what is causing it but I think it might be the ordering of the items when the table is created?

Reply:
That may be a bug in react-table (it's kinda volatile right now). I'll track it down.